requirements.txt

```diff
@@ -1,3 +1,2 @@
 schematics==2.1.0
 six==1.12.0
-twine==1.13.0
```

Comment:
This is needed for pypi publishing. Instead of including it here we could also add it directly to the file in `.circleci/`. Up to you.

Reply:
I have no idea how this happened. I'm guessing indy develop removed this